zero/meta-arducam/recipes-multimedia/arducam-pivariety/files/arducam_pivariety.pc

```diff
@@ -0,0 +1,19 @@
+
+# this template is filled-in by CMake `configure_file(... @ONLY)`
+# the `` are filled in by CMake configure_file(), 
+# from variables set in your CMakeLists.txt or by CMake itself
+#
+# Good tutoral for understanding .pc files: 
+# https://people.freedesktop.org/~dbn/pkg-config-guide.html
+
+prefix="/usr"
+exec_prefix="${prefix}"
+libdir="${prefix}/lib"
+includedir="${prefix}/include"
+
+Name: arducam_pivariety
+Description: Arducam pivariety camera library.
+URL: https://github.com/ArduCAM/Arducam-Pivariety-V4L2-Driver
```

Comment:
Yes, you are right.
I think it's should be points to `https://github.com/ArduCAM/arducam_ppa`.
Will update URL and description.

Reply:
But, I will kept it because this is taken from https://github.com/ArduCAM/arducam_ppa.

> BTW, if the files are taken from someother places, you don't need to fix it. But please write it down where you've got them from.